core/templates/dev/head/header_css_libs.html

```diff
@@ -26,4 +7,6 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 <link rel="stylesheet" type="text/css" media="screen"
-      href="/third_party/static/font-awesome-4.4.0/css/font-awesome.min.css">
+      href="/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
A thought: perhaps manifest.json could specify unminified files as well (to use in dev)?

Also, would the user have to restart the server each time they modify the manifest or gulpfile? Would it be possible to watch for that and recompile, instead?